src/Entity/PropertyId.php

```diff
@@ -64,7 +70,7 @@ public function getEntityType() {
 	 * @return string
 	 */
 	public function serialize() {
-		return json_encode( array( 'property', $this->serialization ) );
+		return json_encode( array( 'property', $this->getSerialization() ) );
```

Comment:
What is the benefit of basically reverting the inlining we did here (and the same in other lines of code in this patch) and calling the getters again?